operations/README.md

```diff
@@ -857,3 +858,10 @@ these backups.
 - [TAPMS Tenant Status API](../api/tapms-operator.md)
 - [Global Tenant Hooks](multi-tenancy/GlobalTenantHooks.md)
 - [Example Workflow](multi-tenancy/ExampleWorkflow.md)
+
+## iSCSI SBPS (Scalable Boot Content Projection Service)
+
+Below are the documents related to iSCSI based boot content projection.
```

Comment:
This will finish resolving the problem documented in https://github.com/Cray-HPE/docs-csm/pull/6132#discussion_r2231898845
```suggestion
Below are the documents related to iSCSI-based boot content projection.
```